solutions/microfrontends/packages/acme-design-system/package.json

```diff
@@ -4,45 +4,69 @@
   "repository": "https://github.com/vercel/examples.git",
   "license": "MIT",
   "scripts": {
-    "lint": "TIMING=1 eslint \"**/*.{ts*,js,jsx}\" --fix",
-    "clean": "rm -rf .turbo && rm -rf node_modules"
+    "dev": "pnpm build:swc -w",
+    "build": "rm -rf dist && pnpm build:swc && pnpm types",
+    "build:swc": "pnpm copy-tailwind && swc src -d dist",
+    "copy-tailwind": "mkdir -p dist && cp src/tailwind.cjs dist/tailwind.cjs",
+    "types": "tsc --emitDeclarationOnly",
+    "lint": "TIMING=1 eslint \"**/*.{ts,tsx,js,jsx}\" --fix",
+    "clean": "rm -rf ./dist && rm -rf .turbo"
   },
   "sideEffects": false,
   "type": "module",
   "exports": {
-    "./tailwind": {
-      "require": "./src/tailwind.cjs"
-    },
-    ".": "./src/index.tsx",
+    ".": "./dist/index.js",
     "./*": [
-      "./src/*.tsx",
-      "./src/*.ts",
-      "./src/*/index.ts"
-    ]
+      "./dist/*.js",
+      "./dist/*/index.js"
+    ],
+    "./tailwind": {
+      "require": "./dist/tailwind.cjs"
+    }
   },
+  "files": [
+    "dist"
+  ],
   "typesVersions": {
     "*": {
       "*": [
         "src/*",
         "src/*/index"
+      ],
+      "tailwind": [
+        "dist/tailwind.d.cts"
       ]
     }
   },
-  "files": [
-    "src"
-  ],
+  "publishConfig": {
+    "typesVersions": {
+      "*": {
+        "*": [
+          "dist/*.d.ts",
+          "dist/*/index.d.ts"
+        ],
+        "tailwind": [
+          "dist/tailwind.d.cts"
+        ]
+      }
+    }
+  },
   "peerDependencies": {
-    "next": "^12.1.0",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0"
+    "next": "*",
+    "react": "^17.0.0 || ^18.0.0-0",
+    "react-dom": "^17.0.0 || ^18.0.0-0"
   },
   "dependencies": {
+    "@swc/helpers": "^0.4.12",
     "clsx": "^1.1.1"
   },
   "devDependencies": {
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.3.10",
     "@types/react": "latest",
+    "chokidar": "^3.5.3",
     "eslint": "^8.16.0",
-    "eslint-config-acme": "*",
-    "typescript": "^4.5.5"
+    "eslint-config-acme": "workspace:^0.0.1",
```

Comment:
I prefer here (and other occurrences) `workspace:*`, see turbo [example](https://github.com/vercel/turbo/blob/main/examples/with-pnpm/apps/web/package.json) & [docs](https://pnpm.io/workspaces#publishing-workspace-packages)

Reply:
Done!